docs/feature_flags.md

```diff
@@ -222,8 +230,10 @@ Examples of equivalent durations:
 * `1h / 2` is the equivalent to `30m` or `1800s`
 * `4h % 3h` is the equivalent to `1h` or `3600s`
 * `(2 ^ 3) * 1m` is the equivalent to `8m` or `480s`
+* `step() + 1` is the equivalent to the query step width increased by 1s.
+* `max(step(), 5s)` is the equivalent to the maximum value between the query step width and `5s`.
+* `min(2 * step() + 5s, 5m)` is the equivalent to the minimum value between the query step increased by `5s` and `5m`.
```

Comment:
Another nit: This whole section (including the old lines) should be changed. It's either "x is equivalent to y" or "x is the equivalent of y". I propose the former as it is shorter.